### DISCUSSION TITLE
I'm running the SharePoint Moderation Scanner Tool + The property or field 'Url' has not been initialized

### DISCUSSION THREAD

**Discussion by PapaJaiswal** [pnp/modernization#564] [Q&A]
Hi Team,

I'm fairly new to SharePoint altogether.

My need is to collect the WFs report from SharePoint Online (Classic WFs Report for SharePoint 2010 & 2013 WFs)
I'm running the SharePoint Modernization Scanner Tool and it gives error for majority of the sites as  '_The property or field 'Url' has not been initialized. It has not been requested or the request has not been executed. It may need to be explicitly requested._'

I tried looking for an answer online on blogs, post etc but no luck

Can someone please guide me

**Comment by jansenbe** [comment]
Hmm, weird. Ensure you use version 2.18 or later in combination with Azure AD App-Only authentication. If it still happens please log an issue in the issue list
